webserver/views/index.py

```diff
@@ -41,3 +45,33 @@ def faq():
 @index_bp.route("/roadmap")
 def roadmap():
     return render_template("index/roadmap.html")
+
+
+# listen-group    listens                        0   282202115       282202115       0               none
+@index_bp.route("/current-status")
+def current_status():
+
+    load = "%.2f %.2f %.2f" % os.getloadavg()
+
+    process = subprocess.Popen([config.KAFKA_RUN_CLASS_BINARY, "kafka.tools.ConsumerOffsetChecker",
```

Comment:
This is a good start, but I'm wary about spinning up all of java each time someone loads this page. Have you looked if there is a way of getting the data from python?